.circleci/config.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           command: yarn lerna publish from-git --create-release=github --yes
       - run:
           name: Create CLI binaries
-          command: npx pkg --out-path ./cli-binaries --targets linux,macos ./packages/cli/
+          command: npx pkg --out-path ./cli-binaries ./packages/cli/
```

Comment:
This should be the harmless change, since those are default options according to https://github.com/vercel/pkg#targets